File: problemas/1000/1020.js
let dias = require('fs').readFileSync('/dev/stdin', 'utf-8');

console.log(`${Number(dias / 365)} ano(s)`);
dias %= 365;
console.log(`${Number(dias / 30)} mes(es)`);
dias %= 30;
console.log(`${dias} dia(s)`);

File: problemas/1000/1020.dart
import 'dart:io';

void main() {
  int dias = int.parse(stdin.readLineSync());
  stdout.writeln('${dias ~/ 365} ano(s)');
  dias %= 365;
  stdout.writeln('${dias ~/ 30} mes(es)');
  dias %= 30;
  stdout.writeln('$dias dia(s)');
}